dts/arm/st/stm32l0-pinctrl.dtsi

```diff
@@ -21,6 +21,12 @@
 					tx = <STM32_PIN_PA2 (STM32_PINMUX_ALT_FUNC_4 | STM32_PUSHPULL_NOPULL)>;
 				};
 			};
+			lpuart1_pins_a: lpuart1@0 {
```

Comment:
Unfortunately, pin generation is not there yet, so to be actually tested, updating boards pinmux.c will be required for now.

Reply:
I've done that in my working copy to test. Nucleo-L073RZ is good for such testing, because LPUART1 is muxed with USART2. I don't think however that  some board pinmux should be modified permanently.